packages/medusa/src/api-v2/admin/users/me/route.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  ContainerRegistrationKeys,
+  MedusaError,
+  remoteQueryObjectFromString,
+} from "@medusajs/utils"
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+} from "../../../../types/routing"
+
+export const GET = async (
```

Comment:
thought(non-blocking): Would it make sense to throw a 401 on both undefined `app_metadata.user_id` and non-existing user? Otherwise, we would be leaking information about the user base no? Theoretically impossible for an adversary to do anything about it, but thought I'd bring it up

Reply:
This route is already protected by the auth middleware that will throw a 401, these guards will probably never be hit. Can still throw a 401 here if it still makes sense, wdyt?